cloudinit/sources/DataSourceScaleway.py

```diff
@@ -410,11 +410,11 @@ def get_hostname(self, fqdn=False, resolve_ip=False, metadata_only=False):
 
     @property
     def availability_zone(self):
-        return None
+        return self.metadata["zone"]
```

Comment:
Are there cases where "zone" may not be set in metadata? If so, we may want to use `self.metadata.get("zone")` instead to avoid a traceback at runtime.